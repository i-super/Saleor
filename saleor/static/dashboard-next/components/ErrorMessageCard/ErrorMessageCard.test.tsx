import * as React from "react";
import * as renderer from "react-test-renderer";

import ErrorMessageCard from "./ErrorMessageCard";

describe("<ErrorMessageCard />", () => {
  it("renders properly", () => {
    const component = renderer.create(
      <ErrorMessageCard message="Things are terrible." />
    );
    expect(component).toMatchSnapshot();
  });
});
