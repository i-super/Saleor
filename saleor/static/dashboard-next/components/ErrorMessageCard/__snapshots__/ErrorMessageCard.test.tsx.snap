// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorMessageCard /> renders properly 1`] = `
<div
  className="MuiPaper-root-2 MuiPaper-elevation2-6 MuiPaper-rounded-3 MuiCard-root-1"
>
  <div
    className="MuiCardContent-root-29"
  >
    <h2
      className="MuiTypography-root-30 MuiTypography-headline-35"
    >
      Error
    </h2>
    <p
      className="MuiTypography-root-30 MuiTypography-body1-39"
    >
      Things are terrible.
    </p>
  </div>
</div>
`;
